pages/api/issue/seo/[...ids].ts

```diff
@@ -1,59 +0,0 @@
-import {NextApiRequest, NextApiResponse} from "next";
```

Comment:
This api effectively returns the url for the image seo -- if this is removed, then `seo.tsx` (the component) needs to fetch this image from somewhere, but such a change was not made

Reply:
The route that generates the image is `web-network\pages\api\seo\[...ids].ts`. This one just returns the title, githubId and body